### PR TITLE
Fixes API pagination of imported entries.

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey @etagwerker, @schmierkov, 

This PR fixes the import of entries and adds some coverage, we were only retrieving the first page of results, at some point we were using a branch instead of the latest version, see: https://github.com/timcraft/freckles/pull/1. We're now using the solution in https://github.com/timcraft/freckles/pull/1#issuecomment-196798160. 

Please check it out, thanks!

### CODE REVIEW

File: app/services/freckle_service.rb [hunk 1]
```diff
@@ -25,9 +25,10 @@ def self.import_entries(start_date, end_date)
     result = client.get_entries(from: start_date, to: end_date)
     save_entries_for(result)
 
-    if result.try(:last_page)
-      last_page = result.last_page.match(/page=(\d+)/)[1].to_i
-      [2..last_page].each do |page|
+    if last = result.try(:link).try(:last)
+      last_page = last.match(/page=(\d+)/)[1].to_i
+      pages = (2..last_page)
```
Comment (schmierkov) [inline app/services/freckle_service.rb]: @mauro-oto why did you add the `pages` variable here? It's pretty small and gets just used once.

Comment (mauro-oto) [inline app/services/freckle_service.rb]: @schmierkov oops, yeah, I don't know why. Thanks!
